docs/nodes/light-node.md

```diff
@@ -17,7 +17,7 @@ The following hardware minimum requirements are recommended for running the ligh
 * Memory: 8 GB RAM
 * CPU: Quad-Core
 * Disk: 5 GB SSD Storage
-* Bandwidth: 1 GB of input-output connection
+* Bandwidth: 1 Gbps for Download/100 Mbps for Upload
```

Comment:
Is this actually needed for a light node? I guess it's just copy-pasted for now, but...the requirements here should be a cell phone and dial-up!

Reply:
I think reality is that DASing plus maintaining connections to sufficient fullnodes is taxing on the bandwidth requirements.

Reply:
@YazzyYaz the requirements should be something like 2 GB or RAM and 56 kbps up and down.

Reply:
ok sounds good, will fix

Reply:
@adlerjohn updated the doc